java/README.md

```diff
@@ -1,111 +1,187 @@
+Valkey General Language Independent Driver for the Enterprise (GLIDE), is an AWS-sponsored, open-source Valkey client that includes support for open-source Redis 6.2 to 7.2.
+Valkey GLIDE works with any distribution that adheres to the Redis Serialization Protocol (RESP) specification, including Amazon ElastiCache, and Amazon MemoryDB.
+Strategic mission-critical applications have requirements for security, optimized performance, minimal downtime, and observability.
+Valkey GLIDE is designed to provide a client experience that helps meet these objectives. It is sponsored and supported by AWS, and comes pre-configured with best practices learned from over a decade of operating RESP-compatible services used by hundreds of thousands of customers.
+To help ensure consistency in development and operations, Valkey GLIDE is implemented using a core driver framework, written in Rust, with extensions made available for each supported programming language.
+This design ensures that updates easily propagate to each language and reduces overall complexity. In this Preview release, Valkey GLIDE is available for Python and Java, with support for Javascript (Node.js) actively under development.
 
 # Getting Started - Java Wrapper
 
+## System Requirements
+
+The beta release of GLIDE for Valkey was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7 (aarch64-apple-darwin).
+
+## Layout of Java code
 The Java client contains the following parts:
 
-1. `client`: A Java-wrapper around the rust-core client.
-2. `examples`: An examples app to test the client against a Redis localhost
-3. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other Java clients.
-4. `integTest`: An integration test sub-project for API and E2E testing
+1. `src`: Rust dynamic library FFI to integrate with [GLIDE core library](../glide-core/README.md).
+2. `client`: A Java-wrapper around the GLIDE core rust library and unit tests for it.
+3. `examples`: An examples app to test the client against a Valkey localhost.
+4. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Valkey and other Java clients.
+5. `integTest`: An integration test sub-project for API and E2E testing.
 
 ## Supported Engine Versions
 
 Refer to the [Supported Engine Versions table](https://github.com/aws/glide-for-redis/blob/main/README.md#supported-engine-versions) for details.
 
 ## Installation and Setup
 
-### Install from Gradle
-
-At the moment, the Java client must be built from source.
-
-### Build from source
+#### Prerequisites
 
-Software Dependencies:
+For developers, please refer to Java's [DEVELOPER.md](./DEVELOPER.md) for further instruction on how to set up your development environment.
```

Comment:
This link doesn't work. There is no "DEVELOPER.md" file. We should resolve this.

I would expect much of the below to move into the "DEVELOPER.md" file as that is all developer centric.

I would expect that a user for their application not need to build from scratch but rather grab from Maven Central or PyPy and that's what the Getting Started in here should focus on.

Reply:
I think this link should work once the two relevant branches are merged. On this branch, the `DEVELOPER.md` file does not yet exist.

Reply:
Confirmed. I just tried the link on the `main` branch and it is working correctly.